source/mainnet/tools/concordium-client.rst

```diff
@@ -61,28 +61,43 @@ Run Concordium Client from the command line. On macOS or Linux, access the comma
 
    .. tab-item:: Windows
 
-      On Windows, to run Concordium Client you have to specify the full path to the executable file
-      (unless you are running from the same directory).
+      On Windows, to run Concordium Client you usually have to specify the full path to the executable file,
+      unless you are running the command from the same directory where ``concordium-client.exe`` is located.
+      In that case, you can invoke the client using either:
+
+      .. code-block:: console
+
+         concordium-client.exe
+
+      or
+
+      .. code-block:: console
+
+         ./concordium-client.exe
+
+      depending on the terminal you are using (e.g. PowerShell, Git Bash, or Command Prompt).
```

Comment:
Note, it's not necessary to specify the `.exe` part on Windows. `concordium-client` is enough. I'm not sure that it's more helpful and confusing to have both options - I expect users that are using `bash` as their shell on Windows can figure it out.